client/blocks/upsell-nudge/README.md

```diff
@@ -0,0 +1,40 @@
+UpsellNudge (JSX)
+===
+
+`UpsellNudge` is a wrapper for the `Banner` component, used to display upsell nudges for plans, domains, services, and product features.
+
+## Usage
+
+```js
+import UpsellNudge from 'blocks/upsell-nudge';
+
+function myUpsell() {
```

Comment:
A nice follow up would be to add this example to the [dev docs](https://wpcalypso.wordpress.com/devdocs/blocks)